CONTRIBUTING.md

```diff
@@ -155,15 +156,30 @@ To maintain consistency between notebooks, please follow the directory structure
 
 In case of output provided by Notebook please create folder ```output``` on the same level as readme file.
 
+### Notebook utils
+
+The _notebook_utils.py_ file in the _notebooks/utils_ directory contains utility functions and classes that can be reused across
+notebooks. It contains a `download_file()` function that optionally shows a progress bar, and a standard way to convert
+segmentation maps to images and display them. The Python file is generated from _notebook_utils.ipynb_ notebook in the same directory.
+If you want to add a function or class to _notebook_utils.py_, please add it to the notebook, and generate the
+Python file with `jupyter nbconvert notebook_utils.ipynb --TagRemovePreprocessor.remove_cell_tags=hide --to script`
+Add a "hide" tag to any demo cells (from the right side gear sidebar) to prevent these cells from being added to the script.
```

Comment:
@helena-intel, Seems like there are unrelated changes

Reply:
@dkurt I rebased to main, the changes should all be from that. I'll doublecheck before merging.